### PR TITLE
Merge sort

### PR DESCRIPTION
**Pull request by jmcclena94**

### CODE REVIEW

File: src/test_merge_sort.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# coding=utf-8
+
```
Comment (nhuntwalker) [inline src/test_merge_sort.py]: You need a few more tests than just these two. Empty lists? Repeated numbers? Consider some more edge cases.

File: src/merge_sort.py [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+    def get_unordered_list():
+        """Call merge_sort on unordered list."""
+        merge_sort(unordered_list)
+
+    def get_reversed_list():
+        """Call merge_sort on reversed list."""
+        merge_sort(reversed_list)
+
+    print('The Merge Sort function sorts a list item in comparison to the'
```
Comment (nhuntwalker) [inline src/merge_sort.py]: You should specifically point out your best and worst cases.
